### PR TITLE
fix(FEC-10161): add kava analytics url from server response

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Issue: kava reports to production env instead QA
Solution: add service protocol for default only.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/kava.js [hunk 1]
```diff
@@ -70,6 +71,7 @@ class Kava extends BasePlugin {
     super(name, player, config);
     this._rateHandler = new KavaRateHandler();
     this._model = new KavaModel();
+    this._serviceUrl = Utils.Http.protocol + '//' + this.config.serviceUrl.replace(/(^\w+:|^)\/\//, '') + SERVICE_PATH;
```
Comment (dan-ziv) [inline src/kava.js:74]: Use template strings since its much more readable

**Review by dan-ziv — CHANGES_REQUESTED** [review]

File: src/kava.js (hunk 1 above)
Comment (OrenMe) [reply src/kava.js:74]: why force the protocol? we assume that a working path is given and nobody promises that SERVICE_PATH is required.
Config should allow getting a fully functional URL as the service url can be proxied by another service fr example in an onPrem env.
The default protocol can be left // so we get the one the page has

**Review by OrenMe — CHANGES_REQUESTED** [review]

**Comment by Yuvalke** [comment]
@OrenMe it's our Kava analytics you can't set there URL for another analytics service.
for example, the value we get from the server doesn't have the SERVICE_PATH, so it won't work without that if I understand it correctly.

**Comment by OrenMe** [comment]
the plugin config should be explicit - it refers to a url, not partial.
I would want us to leave it like that so if issue arrises it will be as easy as setting a new URL.
This means - leave default as is, and pass the correct url in kaltura player - add the service path there.
If someone needs to set another url, let's say testing purposes or setting a local proxy he will be able to set his own url.

**Review by OrenMe — APPROVED** [review]

**Review by dan-ziv — APPROVED** [review]

**Pull request merged**